packages/components/psammead-navigation/README.md

```diff
@@ -71,6 +101,30 @@ import { latin } from '@bbc/gel-foundations/scripts';
 </Navigation>;
 ```
 
+## Dropdown Navigation Usage
+
+```jsx
+import React from 'react';
+import { CanonicalDropdown, DropdownNavigationLi } from '@bbc/psammead-navigation';
```

Comment:
Do we need to export these from the index file to be able to import them like this?

Reply:
Ah Alistair already mentioned this, sorry!